Dockerfile

```diff
@@ -45,11 +48,6 @@ LABEL org.label-schema.vcs-url="https://github.com/MrSaints/kubeseal-web" \
 
 RUN apk add --no-cache bash ca-certificates curl jq wget nano
 
-RUN wget https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.17.3/kubeseal-0.17.3-linux-${TARGETARCH}.tar.gz -O kubeseal.tar.gz \
-    && tar -xzf kubeseal.tar.gz -C /tmp/ \
-    && install -m 755 /tmp/kubeseal /usr/local/bin/kubeseal \
-    && rm -rf kubeseal.tar.gz /tmp/kubeseal
```

Comment:
Why is this removed?

Reply:
Woops accidentally removed by mistake, oversight from my end